docs/elements.md

```diff
@@ -1241,18 +1241,19 @@ def generate(data):
+| `show-dtype`            | boolean       | false    | Show the data types contained in each column of the DataFrame at the bottom of each column. Types used correspond to the `display-language` parameter.                                                                                                                                                                            |
+| `display-language`      | "python", "r" | "python" | Language to use for displaying data types and indices.                                                                                                                                                                                                                                                                            |
+| `display-variable-name` | string        | "df"     | Variable name to display in code to recreate DataFrame.                                                                                                                                                                                                                                                                           |
+| `show-python`           | boolean       | true     | Show code that can be used to recreate the DataFrame in Python in a separate tab.                                                                                                                                                                                                                                                 |
+| `digits`                | integer       | -        | Number of digits to display for floating point entries.                                                                                                                                                                                                                                                                           |
+| `width`                 | integer       | 500      | Max characters per line for displaying Python code.                                                                                                                                                                                                                                                                               |
+| `presentation-type`     | string        | `'g'`    | Floating point number display format. If `digits` is specified, formatted using `'{:.{digits}{presentation-type}}'`, otherwise `'{:{presentation-type}}'`. See [the Python documentation](https://docs.python.org/3/library/string.html#format-specification-mini-language) for more information on allowable presentation types. |
```

Comment:
That makes sense. I want to avoid putting the full list because I think the majority of people who use this will have a format code in mind they want (i.e. `f` or one of the more common ones), and the docs for that are a little long.